ni_measurementlink_service/session_management/_reservation.py

```diff
@@ -527,6 +715,42 @@ def initialize_session(
         """
         return self._initialize_session_core(session_constructor, instrument_type_id)
 
+    def initialize_multiplexer_session(
+        self,
+        session_constructor: Callable[[MultiplexerSessionInformation], TMultiplexerSession],
+        multiplexer_resource_name: Optional[str] = None,
+        multiplexer_type_id: Optional[str] = None,
```

Comment:
Thanks for explaining why you added a multiplexer_resource_name parameter.
 
I think you should design these API to encourage using multiplexer_type_id and discourage using multiplexer_resource_name by doing both of the following:
- Swap the parameters so that multiplexer_type_id can be passed as a positional parameter.
- Make multiplexer_resource_name a keyword-only parameter so that clients that want it must ask for it by name.

Reply:
Sure. This will also enable us to achieve some level of consistency with the other existing APIs.